src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -865,6 +870,39 @@ private static TypedConstant GetParamArrayArgument(ParameterSymbol parameter, Im
             return new TypedConstant(parameter.Type, values.AsImmutableOrNull());
         }
 
+        private static bool TryGetNormalParamValue(ParameterSymbol parameter, ImmutableArray<TypedConstant> constructorArgsArray,
+            int argIndex, Conversions conversions, out TypedConstant result)
+        {
+            TypedConstant argument = constructorArgsArray[argIndex];
+            if (argument.Kind != TypedConstantKind.Array)
+            {
```

Comment:
Do we hit this case with `Mark(a: true, b: null)`? (We should generate a `null` array.)